crates/uv/src/commands/project/remove.rs

```diff
@@ -223,18 +223,20 @@ enum Target {
 ///
 /// This is useful when a dependency of the user-specified type was not found, but it may be present
 /// elsewhere.
-fn warn_if_present(name: &PackageName, pyproject: &PyProjectTomlMut) {
-    for dep_ty in pyproject.find_dependency(name) {
+fn warn_if_present(requirement: &PackageName, pyproject: &PyProjectTomlMut) {
```

Comment:
Can you revert the variable changes here where it's a `PackageName` again?

Reply:
Done